src/content/article/over-a-year-late-a-dollar-short-more-than-two-decades-of-kodi/index.md

```diff
@@ -0,0 +1,87 @@
+
+And we’ve stayed volunteer. As a consumer-oriented open source project, no company has ever sponsored more than a port or a feature; indeed, since the nonprofit foundation behind it isn’t setup to be engaged, at worst companies have simply given some money to an individual to work on a feature. Even this is definitely not the norm: 99.9% of Kodi has been built out of passion and love for multimedia and the desire to have an open source platform for which people can decide the look and feel and customize to their liking, in both hardware and software terms. 
+
+I do have to call out our longest and biggest sponsor/partner has been [Flirc](https://flirc.tv), their support with this project has been unwavering since the initial case.
+
+This is truly software on your own terms, and to me, is the essence of open source.
+
+**Wrapping it All Up**
+
+Sorry this became so wordy, but I’ve been wanting to write a ’20 year blog post’ for so long, and it’s now been nearly 22 years since XBMP binaries first became available. Time flies. I hope this blog helped you stop for a moment and marvel at how far we’ve came, even if you’re only been a user since we’ve been called Kodi: the developers see themselves in you and you're the reason why they've kept hacking on this project for so long.
+
+So, let’s raise our glasses for over 20 years, and, yes, we still have MANY lines of original, first generation code! We have to thank everyone who’s been involved and supported us: everyone who submitted a pull request or posted a message on the forum, all the devs who have tirelessly worked on the ‘dependency’ libraries that mean so much to us, and especially the official Team XBMC and Kodi Foundation developers. You folks, who continue to iterate, tweak and build the future of this platform, enable millions to consume media without having to care about how it works and why.
+
+![Kodi Home Screen, 2014](/images/blog/kodi-2014-1080p.webp "Image of the 2014 Kodi home screen and menu")
+
+And while you’re here, we made a T-shirt to commemorate this moment! It (hopefully) has every team member on it. As we said, we’re volunteers, and the only money our nonprofit receives is thru donations, merchandise and the only licensed product we’ve ever been a part of, Flirc’s Kodi Raspberry Pi case.
+
+Take your pick from:
+
+* [T-shirt dark](https://kodi-t-shirt-store.creator-spring.com/listing/kodi-nexus-dark?product=46&variation=2752)
+* [T-shirt light](https://kodi-t-shirt-store.creator-spring.com/listing/kodi-nexus-light?product=46&variation=2750)
+* [Pillow](https://kodi-t-shirt-store.creator-spring.com/listing/kodi-nexus-light?product=585)
+
+.. or the whole store: https://kodi-t-shirt-store.creator-spring.com/
+
+Please, purchase something, and help us to keep thriving for the next 20 years; I personally plan our Devcons, and we might not be able to get together in 2025 without help from viewers like you, so anything you can do to help, we appreciate you.
```

Comment:
```suggestion
Please, purchase something, and help us to keep thriving for the next 20 years; I personally plan our Devcons, and we might not be able to get together in 2025 without help from readers like you, so anything you can do to help, we appreciate you.
```
Or maybe just "people"?

Reply:
I pondered this one - I agree that "readers", well, reads better, but I think the intent was "people who view stuff on Kodi". Split the difference: _people like you_.